libguard/guard_interface.cpp

```diff
@@ -329,6 +329,36 @@ void clearAll()
     }
 }
 
+void clearInvalidRecords()
```

Comment:
is it invalid or resolved?

Reply:
I am not sure why we can call it as resolved or not. Basically I have seen that in reconfig look hostboot sets the record ID as F..F, so not sure to call it resolved.

Reply:
I don't know the intention of updating ID as `0xFFFFFFFF` - I heard like if guarded hardware is repair or replaced then id will get change as `0xFFFFFFFF` so, we are saying it as resolved. if you have seen some other use cases then bring in some forum or pick whatever makes sense to you.

Reply:
Will change to resolved.